src/apis/btc-core.ts

```diff
@@ -142,6 +165,27 @@ export class DefaultBTCCoreAPI implements BTCCoreAPI {
         return this.getData(this.txApi.getTxMerkleBlockProof(txid));
     }
 
+    getTx(txid: string): Promise<Transaction> {
+        return this.getData(this.txApi.getTx(txid));
+    }
+
+    async getUtxoAmount(txid: string, recipient: string): Promise<string> {
+        let amount = "0";
```

Comment:
this should be a `number`

Reply:
is `vout.value` a `number`? Just checking since it might be a string due to the limited `number` size

Reply:
It is a number